src/internal/mod.rs

```diff
@@ -102,6 +103,9 @@ quick_error! {
         NotGroupAdmin(group_id:group_api::GroupId) {
             display("You're are not an administrator of group '{}'", group_id.0)
         }
+        InvalidPolicy {
+            display("A policy must specify one or more of category, sensitivity, data_subject, substitute_id")
```

Comment:
Pretty minor, but I'd remove the underscores from these names in this error

Reply:
I removed the whole error as it's not needed.